cloud-shell/Dockerfile

```diff
@@ -20,5 +20,13 @@ RUN sudo apt-get install unzip && \
   unzip -o terraform.zip && \
   sudo install terraform /usr/local/bin
 
+# install python libraries
+RUN python3 -m pip install google-cloud-pubsub
+RUN python3 -m pip install click
+RUN python3 -m pip install google-cloud-monitoring
+
+# set env var
+RUN echo "VERSION=v0.2.5" >> /etc/environment
```

Comment:
you probably want to set the version to "latest" or something else that means "master branch". Then in the release script it will be updated to whatever tag we use to make a release.

Reply:
Okay, I see. I will look into doing this.

Reply:
This is for metrics purposes, right? Do we want "latest" to show up in our metrics? Or just the last released version?

Either way, we'll have to update the make-release script to make sure this value is updated